### PR TITLE
Fix core dump during test

### PR DESCRIPTION
**Pull request by Luni-4**
I know this is an horrible hack, but at least it works :)

### CODE REVIEW

File: bubblesub/api/video.py [hunk 1]
```diff
@@ -24,11 +24,11 @@
 import typing as T
 from pathlib import Path
 
-import ffms2
 import numpy as np
 import PIL.Image
 from PyQt5 import QtCore
 
+import ffms2
```
Comment (rr-) [inline bubblesub/api/video.py]: This is third party, I can understand that `isort` detect this incorrectly.
Let's add an exception here: https://github.com/rr-/bubblesub/blob/master/pyproject.toml#L6 and move it back.

File: bubblesub/tests/api/test_video.py [hunk 2]
```diff
@@ -36,7 +35,7 @@ def _test_align_pts_to_frame(
     ) as video_api_mock:
         video_api_mock.return_value = [0, 10, 20]
 
-        video_api = VideoApi(log_api, subs_api)
+        video_api = VideoApi(log_api, subs_api, True)
```
Comment (rr-) [inline bubblesub/tests/api/test_video.py]: Let's call it `shell=True`.

**Comment by Luni-4** [comment]
> TBH I understand the need for this change, but I don't like it, and I don't like the alternative (starting workers from a dedicated method), since it'd decompose object initialization to two phases…

I guess the problem is related to this one:  [https://stackoverflow.com/questions/13662209/segmentation-fault-core-dumped-executing-a-thread](url)

> Can we do something about the crash? Do you have any details?

Tested with: `valgrind pytest test_video.py`

`==26407== Process terminating with default action of signal 6 (SIGABRT)`
`==26407==    at 0x48B2077: raise (raise.c:51)`
`==26407==    by 0x4893534: abort (abort.c:79)`
`==26407==    by 0x8D2E3BB: QMessageLogger::fatal(char const*, ...) const (in /usr/local/lib/python3.7/dist-packages/PyQt5/Qt/lib/libQt5Core.so.5)`
`==26407==    by 0x8D4EB7B: QThread::~QThread() (in /usr/local/lib/python3.7/dist-packages/PyQt5/Qt/lib/libQt5Core.so.5)`
`==26407==    by 0x88D7BA8: sipQThread::~sipQThread() (in /usr/local/lib/python3.7/dist-packages/PyQt5/QtCore.so)`
`==26407==    by 0xB984945: forgetObject (in /usr/local/lib/python3.7/dist-packages/PyQt5/sip.so)`
`==26407==    by 0xB984968: sipWrapper_dealloc (in /usr/local/lib/python3.7/dist-packages/PyQt5/sip.so)`
`==26407==    by 0x576380: subtype_dealloc.lto_priv.2372 (typeobject.c:1256)`
`==26407==    by 0x59CC70: dict_dealloc.lto_priv.195 (dictobject.c:1905)`
`==26407==    by 0xB975626: sipSimpleWrapper_clear (in /usr/local/lib/python3.7/dist-packages/PyQt5/sip.so)`
`==26407==    by 0xB979033: sipWrapper_clear (in /usr/local/lib/python3.7/dist-packages/PyQt5/sip.so)`
`==26407==    by 0xB984970: sipWrapper_dealloc (in /usr/local/lib/python3.7/dist-packages/PyQt5/sip.so)`

Unfortunately, I can't test with `gdb`

File: bubblesub/api/video.py (hunk 1 above)
Comment (Luni-4) [reply bubblesub/api/video.py]: Fixed manually, for now. I agree with the addition of an exception

**Comment by rr-** [comment]
As per our discussion, I'll leave this pr open, as a reminder for myself to find a proper fix for the crash that shouldn't be happening even if we do start the thread from inside the shell.

**Pull request closed without merging**